serokell-style.md

```diff
@@ -1,27 +1,28 @@
 Serokell Haskell Style Guide
 ============================
 > This style guide is mostly a copy of [Johan Tibell's guide](https://github.com/tibbe/haskell-style-guide/blob/master/haskell-style.md)
-> with some restructurization, elaboration on some topics and some additions.
-> This style guide's aims are code beauty, readability and understandability.
+> with some restructuring, elaboration on some topics and some additions.
+> The aims of this style guide are code beauty, readability and understandability.
 
 You can find our other formatting utilites and guidelines which expand the code style:
-* [.stylish-haskell.yaml](https://github.com/serokell/serokell-util/blob/master/.stylish-haskell.yaml)
-* [Universum](https://github.com/serokell/universum)
-* [Custom _HLint_ rules](https://github.com/input-output-hk/cardano-sl/blob/master/HLint.hs)
 
-General guide lines
+* [`stylish-haskell` config](https://github.com/serokell/serokell-util/blob/master/.stylish-haskell.yaml)
+* [`hlint` config](https://github.com/serokell/serokell-util/blob/master/.hlint.yaml)
+* [`hlint` config](https://github.com/serokell/serokell-util/blob/master/.hlint-universum.yaml) specific to [`universum`](https://github.com/serokell/universum), our custom Prelude
+
+General guidelines
 -------------------
 ### Line Length
 
 Maximum line length is *80 characters* or *100 characters* if necessary.
 
-Modern screens have high definition and big width.
-But with some tiling managers and two terminals on one screen you are not able to
-see many characters on one line.
-On the other hand, restricting line size to a very small number like 80 leads to
-some crazy indentation despite the fact that
-shorter lines should force you to write well structured code.
-That's why *100* is a reasonable compromize.
+Modern screens are wide and have high resolution.
+Yet, when you have two terminals open side-by-side or edit two files simultaneously,
+you will not be able to fit many characters on a line.
+On the other hand, restricting line length to a very small number like 80 leads
+to crazy indentation despite the fact that shorter lines should supposedly force you
+to write well-structured code.
+That is why we pick *100* as a reasonable compromise.
```

Comment:
Or we can use 2 spaces for indentation and keep the 80 character limit.

Reply:
I would say that trying to fit with 80 with a possible hard limit of 100 is a good strategy. I've already pointed out that if you use two-pane 12-inch display (ThinkPad X series) you can only fit about 80 characters even if you have a good sight. Of course, it's not a problem on bigger monitors, but thinkpads are common, and also macbook 12 is too.